modules/timepoint_list/README.md

```diff
@@ -0,0 +1,48 @@
+# Timepoint List
+
+## Purpose
+
+The `timepoint_list` module serves as an entry-point for data entry
```

Comment:
* add: It is also known as the Candidate Profile page. 
* Don't say "dashboard".  `There is only one Dashboard.`

Reply:
@driusan are you planning to address this?

Reply:
I addressed the first part in the paragraph below, and I haven't decide on the second. The sentence is using "dashboard" in the English language definition of the word and doesn't claim to be the dashboard module. There's no reason that there can't be multiple different kinds of dashboard to provide overviews of different features/entities. Calling it a "dashboard" here gives more flexibility to house features that aren't related to listing timepoints (which it already does.)

Reply:
my point is -- it confuses the terminology we're using in Loris, not that the word is incorrect.

(yes, we could call many things dashboards, like the Stats module or the bvl/Img QC modules, but why confuse it?)

This is the Candidate Profile page as well as the Timepoint List, there's no need to add "dashboard" which is an Actual Thing elsewhere. -- let's wait til the candidate module (replacing this) gets deployed before we call this a dashboard.  
Presumably there will be a lot more dashboard-like info visible in the new page, like: how many outstanding forms? what's the consent status?  how many QCd images?  any biospecimens? etc...